### DISCUSSION TITLE
[SOLVED] Formatting with a real standard

### DISCUSSION THREAD

**Discussion by Shinigami92** [poolifier/poolifier#76] [Show and tell]
@pioardi 

Working together with @jerome-benoit turns out I really have some problems with it's so called "standard"...

`prettier-standard` uses a [outdated](https://github.com/sheerun/prettier-standard/blob/master/package.json#L34) version of `prettierx`
`prettierx` uses a [outdated](https://github.com/brodybits/prettierx/blob/prettierx-0.11.3/package.json#L112) version of `prettier`

And the so called standard is just used by `269325` weekly downloads at the moment were `prettier` and `eslint` are used by ~15 times multiple of millions of weekly downloads

It also generates really strange and in my opinion problematic code (https://github.com/Shinigami92/poolifier/pull/1#discussion_r571467475, https://github.com/Shinigami92/poolifier/pull/1#discussion_r571456269)

So should we consider to use standard instead of "standard"?

[<img src="https://user-images.githubusercontent.com/7195563/107128939-e4b55d00-68c1-11eb-84aa-665a1bc6f539.png" width="400px"/>](https://www.npmtrends.com/prettier-vs-eslint-vs-standard-vs-prettier-standard)

<img src="https://user-images.githubusercontent.com/7195563/107129117-6bb70500-68c3-11eb-92ac-b055a319e2ec.png" width="400px"/> <img src="https://user-images.githubusercontent.com/7195563/107129124-7376a980-68c3-11eb-98e9-533db208d4fb.png" width="400px"/>
<img src="https://user-images.githubusercontent.com/7195563/107129132-7a052100-68c3-11eb-8c98-a3d7dd3c10b3.png" width="400px"/> <img src="https://user-images.githubusercontent.com/7195563/107129136-812c2f00-68c3-11eb-9785-e3f59c99c44e.png" width="400px"/>

<img src="https://imgs.xkcd.com/comics/standards.png" width="400px"/>

**Comment by pioardi** [comment]
I am not sure to get your point, we added prettier-standard only to extend standard with the prettier format.
Now if standard is adding async/await keyword we need to understand why of this rule were added to the standard library and if can be problematic for us

**Comment by Shinigami92** [comment]
We successfully moved to native eslint + prettier(x)

As a positive side-effect we have now warnings in code review `changed files` tab :tada:
